ts/IterableInbox.tsx

```diff
@@ -301,7 +304,13 @@ const IterableInbox = ({
             {showMessageList(loading)}
             {showMessageDisplay(rowViewModels, selectedRowViewModelIdx)}
          </Animated.View>
-      </SafeAreaView>
+      )
+   }
+
+   return(
+      (inboxMode === "button") ? 
```

Comment:
See if you can do enums here

Reply:
I will make the change to safeAreaMode as a boolean value.